docs/source/releases/v1.6.rst

```diff
@@ -32,36 +32,46 @@ What's new in Oscar 1.6?
 
 Removal of deprecated features
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
- - Removed ``OSCAR_PROMOTION_MERCHANDISING_BLOCK_TYPES`` setting since
-   merchandising block promotions, which used it, already removed for a while.
+
+ - Removed the redundant ``OSCAR_PROMOTION_MERCHANDISING_BLOCK_TYPES`` setting.
+
+ - Removed the redundant ``OSCAR_PROMOTION_MERCHANDISING_BLOCK_TYPES`` setting.
 
 Minor changes
 ~~~~~~~~~~~~~
- - Majority of the Oscar imports now using dynamic loading, instead of
-   direct importing for better extensibility and customizability, except
-   signals.
- - Customer transaction emails reworked on top of mailgun layout and has
-   better look.
+ - The majority of the Oscar class imports now use dynamic loading, instead of
+   direct importing, for better extensibility and customizability.
+
+ - Customer transaction emails were reworked to use Mailgun templates, so that
+   they have a more modern appearance.
+
  - ``SearchHandler.get_search_form`` method now accepts additional
    keyword arguments, which will be passed on search form class instance
    initiation.
+
  - Added ``get_stock_info`` hook to ``oscar.apps.basket.models.Basket``  for
    implementing strategies that depend on product options.
+
  - Fixed the page create/update views in the dashboard to correctly validate
    URLs. Invalid characters in URLs will raise a validation error, as will
    URLs longer than 100 characters.
- - Default shipping method on basket summary page uses default user shipping
-   address in order to calculate indicative shipping cost. Default shipping
-   address definition can be overridden in
-   ``oscar.apps.basket.views.BasketView.get_default_shipping_address`` method.
+
+ - The default shipping method ( ``oscar.apps.basket.views.BasketView.get_default_shipping_address``)
```

Comment:
Shall we move `oscar.apps.basket.views.BasketView.get_default_shipping_address` in order to keep line length limit up to 79 chars?

Reply:
Done.